week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -31,13 +31,25 @@ var parkAvenueHouse = {
 */
 
 // returns the full name (first name + last name) of the owner of the house
-function getOwnerFullName(house) {}
+function getOwnerFullName(house) {
+  return house.currentOwner.firstName + " " + house.currentOwner.lastName;
+}
 
 // returns an array of the owners' email addresses of the two houses
-function getEmailAddresses(house1, house2) {}
+function getEmailAddresses(house1, house2) {
+  return [house1.currentOwner.email, house2.currentOwner.email];
+}
 
 // returns the address for the cheapest house out of the two
-function getCheapestAddress(house1, house2) {}
+function getCheapestAddress(house1, house2) {
+  var housesPrices = [house1.price, house2.price];
+  var theCheapestPrice = Math.min(...housesPrices);
+  if (house1.price === theCheapestPrice) return house1.address;
+  if (house2.price === theCheapestPrice) return house2.address;

```

Comment:
This does work, but you can do it more simply by comparing house1.price and house2.price directly.